dali/kernels/CMakeLists.txt

```diff
@@ -29,8 +31,10 @@ collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 collect_sources(DALI_KERNEL_SRCS)
 collect_test_sources(DALI_KERNEL_TEST_SRCS)
 
+# cuFFT library
 add_library(${dali_kernel_lib} SHARED ${DALI_KERNEL_SRCS})
 target_link_libraries(${dali_kernel_lib} PUBLIC ${dali_core_lib})
+target_link_libraries(${dali_kernel_lib} PRIVATE ${CUDA_cufft_static_LIBRARY})
```

Comment:
This would likely lead to strange errors, since libcufft_static in different shared objects are totally independent and the cufft handles are incompatible.